app/controllers/api/v1/sessions_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   module V1
     class SessionsController < DeviseTokenAuth::SessionsController
+      skip_before_action :verify_authenticity_token, only: :create
```

Comment:
Instead of doing this, shouldn't you be using here:
```include Api::Concerns::ActAsApiRequest``` ?

Reply:
Yes, thank you, i did that include in the wrong file.